src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -510,15 +518,18 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       stay using finalData
     case Event(PersistenceSucceeded(callKey, ExecutionStatus.Preempted, callOutputs), data) =>
       executionStore += callKey -> ExecutionStatus.Preempted
-      val updatedData = startRunnableCalls(data)
-      val finalData = updatedData.removePersisting(callKey, ExecutionStatus.Preempted)
-      self ! CheckForWorkflowComplete
-      stay using finalData
+      val updatedData = retryCall(callKey, data).removePersisting(callKey, ExecutionStatus.Preempted)
```

Comment:
The last thing I'm not really sure I understand is why we seem to do `doSomething.removePersisting(callKey, ExecutionStatus.Preempted)` a few times in this code